.github/workflows/python2-3-app.yml

```diff
@@ -0,0 +1,43 @@
+name: DB12 testing
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.6, 2.7]
+        
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      
+    - name: Add conda to system path
+      run: |
+        echo $CONDA/bin >> $GITHUB_PATH
+        
+    - name: Set up Conda and Install dependencies
+      run: |
+        conda create -c conda-forge -c free -n python python=${{ matrix.python-version }}
+        conda info --env
+        source /usr/share/miniconda/etc/profile.d/conda.sh
+        conda activate python
+        pip install pytest
+        pip install future
+        pip install pylint
+        
+    - name: Pylint
+      run: |
+        source /usr/share/miniconda/etc/profile.d/conda.sh
+        conda activate python
+        pylint test.py --max-statements=100 --exit-zero
+        pylint DIRACbenchmark.py --exit-zero
```

Comment:
Good, but you also have to follow what pylint is telling you and remove the warnings :-) 
For instance at line 18 in this log: https://github.com/ImanIra0ui/DB12/runs/2900860977?check_suite_focus=true#step:5:18

- `C:  4, 0: Trailing whitespace (trailing-whitespace)` means that there is a tab while the line should be empty.

Here is the list of the messages: http://pylint-messages.wikidot.com/all-messages

Reply:
Oh I see. I'll go ahead and modify the DIRACbenchmark.py code according to those warnings.